RecoTracker/MkFit/plugins/HitSelectionWindowsPhase1.h

```diff
@@ -0,0 +1,5868 @@
+#ifndef RecoTracker_MkFit_plugins_HitSelectionWindowsPhase1_h
+#define RecoTracker_MkFit_plugins_HitSelectionWindowsPhase1_h
+
+namespace mkfit {
+
+  struct HitSelectionWindowsPhase1 {
+  public:
+    float m_dp_params[9][72][3];
+    float m_dq_params[9][72][3];
+    float m_c2_params[9][72][3];
+
+    HitSelectionWindowsPhase1() {
```

Comment:
how did you get this list of "magic" numbers ? :(

Reply:
This is intermediate stage of hit search window parameters, obtained by analysing residuals between sim tracks and hits during building. After being loaded through this parameter set, they are exported as JSON.

We know this is not ideal and will be improving this in the future.

Reply:
> After being loaded through this parameter set, they are exported as JSON.

Just to clarify, the "loading and exporting" are only for the standalone mkFit package, although in principle the exporting functionality could be added for the CMSSW module as well.

Also the (hardcoded) values here and in the JSON files are currently duplicates. Ideally this duplication should be removed, the way to do that depends on the preferred way to provide this information in CMSSW.

Reply:
> Also the (hardcoded) values here and in the JSON files are currently duplicates. Ideally this duplication should be removed, the way to do that depends on the preferred way to provide this information in CMSSW.

Duplication is now removed by removing this file and using the JSON files as the only source of the parameters.